services/app-web/src/components/DataDetail/DataDetail.stories.tsx

```diff
@@ -9,41 +12,64 @@ export default {
             'DataDetail displays definition terms and descriptions using semantic HTML. This is useful for summarizing static data.',
     },
 }
-
-export const WithString = (): React.ReactElement => (
-    <dl>
-        <DataDetail
-            id="rainfall"
-            label="Average rainfall in May"
-            data="31.58 inches"
-        />
-    </dl>
-)
-export const WithAddress = (): React.ReactElement => (
+const Template: Story<DataDetailProps> = (args) => (
     <dl>
-        <DataDetail
-            id="disney"
-            label="Disney World Contact Info"
-            data={
-                <address>
-                    Mickey Mouse
-                    <br />
-                    <a href="mailto:mickey@disney.com">mickey@disney.com</a>
-                    <br />
-                    <a href="tel:555-555-5555">555-555-5555</a>
-                </address>
-            }
-        />
+        <DataDetail {...args} />
     </dl>
 )
 
-export const WithMissingField = (): React.ReactElement => (
-    <dl>
-        <DataDetail
-            id="crystal-ball"
-            label="The secret to the future"
-            data={undefined}
-            explainMissingData
-        />
-    </dl>
-)
+export const WithString = Template.bind({})
+WithString.decorators = [(Story) => ProvidersDecorator(Story, {})]
+WithString.args = {
+    id: 'rainfall',
+    label: 'Average rainfall in May',
+    data: '31.58 inches',
+}
+
+export const WithMissingField = Template.bind({})
+WithMissingField.decorators = [(Story) => ProvidersDecorator(Story, {})]
+WithMissingField.args = {
+    id: 'crystal-ball',
+    label: 'The secret to the future',
+    data: undefined,
+    explainMissingData: true,
+}
+
+export const WithCheckboxList = Template.bind({})
+WithCheckboxList.decorators = [(Story) => ProvidersDecorator(Story, {})]
+WithCheckboxList.args = {
+    id: 'crystal-ball',
+    label: 'The secret to the future',
+    data: ['this', 'that', 'the other thing'],
+}
+
+// Having trouble getting these stories to render without crashing story
```

Comment:
I can't get these stories rendering properly. I am thinking it doesn't like the use of subcomponents as props. I have been referencing [storybook docs ](https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components/)but not getting anywhere. Leaving it here in case anyone sees what I'm missing, otherwise once approve I will make a commit removing these stories all together.

Reply:
I poked around `DataDetailContactField.tsx` and looks like its actually `getActuaryFirm()` causing the Storybook issue.

```
  {isCertainActuaryContact(contact) && (
      <>
          <br />{' '}
          <span>{getActuaryFirm(contact as ActuaryContact)}</span>
      </>
  )}
```

If you move that function into `DataDetailContactField.tsx` everything works fine. I'm not sure exactly why. Possibly not being able to call out that function from storybook?

Reply:
I really think something is broked with our storybook. Subcomponents are not working as documented. I think its something odd going on maybe with imports in our monorepo structure.